modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/wings.dm

```diff
@@ -260,7 +260,6 @@
 /datum/sprite_accessory/wings/moth/burnt
 	name = "Moth (Burnt)"
 	icon_state = "burnt_off"
-	locked = TRUE
```

Comment:
this was intentional, readd it

Reply:
Oh wait. Did i misunderstand?

Reply:
yes, the `locked` var stops you from picking it by default, which is intended here.

Reply:
> yes, the `locked` var stops you from picking it by default, which is intended here.

Oh, then I really misunderstand everything. I'll readd it a bit later and rename PR for only adding wings